core/src/types/ops.rs

```diff
@@ -377,17 +382,6 @@ impl OpWrite {
         self.cache_control = Some(cache_control.to_string());
         self
     }
-
-    /// Set the If-Match of the option
-    pub fn with_if_match(mut self, if_match: &str) -> Self {
-        self.if_match = Some(if_match.to_string());
-        self
-    }
-
-    /// Get If-Match from option
-    pub fn if_match(&self) -> Option<&str> {
-        self.if_match.as_deref()
-    }
```

Comment:
Is it right?

Reply:
Yes, OpWrite doesn't have `if-match` support.